boards/arm/hexiwear_kw40z/Kconfig.defconfig

```diff
@@ -30,19 +30,6 @@ config PINMUX_MCUX_PORTC
 
 endif # PINMUX_MCUX
 
-if GPIO_MCUX
-
-config GPIO_MCUX_PORTA
-	default y
-
-config GPIO_MCUX_PORTB
-	default n
-
-config GPIO_MCUX_PORTC
-	default y if UART_MCUX_LPUART_0 || I2C_1
```

Comment:
another case of losing the conditional enable

Reply:
I'm confused in this case what enabling GPIO_MCUX_PORTC ends up doing.  In the hexiwear_k64 there is at least a call to gpio_pin_configure() for the MAX30101 being enabled.  In the pinmux.c for hexiwear_kw40z there isn't any gpio API calls on portc.

So at most this just seems to get IRQs wired up.